### PR TITLE
Remove runDebugWithAsserts

### PR DESCRIPTION
**Pull request by mincong-h**
Fix issue #1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -181,7 +181,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                         <showDeprecation>true</showDeprecation>
                         <showWarnings>true</showWarnings>
                         <compilerArgs>
-                            <arg>-Werror</arg>
+                            <!-- arg>-Werror</arg -->
```
Comment (danieldietrich) [inline pom.xml]: Is it because of the new deprecations within the Vavr core library?

Comment (mincong-h) [reply pom.xml]: Yes, you are right.

**Comment by mincong-h** [comment]
@danieldietrich, could you enable the build in Travis CI https://travis-ci.org/vavr-io/vavr-benchmark? I would like to see the build working and if it is not, I can fix it this weekend.

**Comment by danieldietrich** [comment]
I will do that but I think the jobs will be terminated if they are not finished after a certain amount of time. The benchmarks run for hours, I think that will not work on Travis-CI. But I will check that!

**Comment by danieldietrich** [comment]
I triggered a build: https://travis-ci.org/vavr-io/vavr-benchmark/builds/628085592

**Comment by mincong-h** [comment]
Oh, I didn't think about the timeout problem. Thanks anyway :)

**Comment by danieldietrich** [comment]
It errored after 50 minutes with a timeout, as I thought.

Could you please change the travis yaml the way that the CI job only compiles the (test) sources without running the benchmarks. That way we ensure that PRs are statically correct. Thx!

**Pull request merged**